src/Map/Map.stories.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react'
+
+import Map from './index'
+import exampleJson from './example.json'
+import colors from '../colors'
+
+export default {
+  title: 'Map Component',
+  component: Map,
+  argTypes: {
+    height: {
+      control: {
+        type: 'text',
+      },
+    },
+    width: {
+      control: {
+        type: 'text',
+      },
+    },
+    startPositionLong: {
+      control: {
+        type: 'number',
+      },
+    },
+    startPositionLat: {
+      control: {
+        type: 'number',
+      },
+    },
+    startPositionZoom: {
+      control: {
+        type: 'number',
+      },
+    },
+    style: {
+      control: {
+        type: 'text',
+      },
+    },
+    cluster: {
+      control: {
+        type: 'boolean',
+      },
+    },
+    clusterMaxZoom: {
+      control: {
+        type: 'number',
+      },
+    },
+    clusterAreaRadius: {
+      control: {
+        type: 'number',
+      },
+    },
+    clusterColor: {
+      control: {
+        type: 'color',
+      },
+    },
+    clusterRadius: {
+      control: {
+        type: 'number',
+      },
+    },
+    countFont: {
+      control: {
+        type: 'array',
+      },
+    },
+    countFontSize: {
+      control: {
+        type: 'number',
+      },
+    },
+    countFontColor: {
+      control: {
+        type: 'color',
+      },
+    },
+    sourceJson: {
+      control: {
+        type: 'text',
+      },
+    },
+  },
+}
+
+const Template = (args) => {
+  const props = {
+    token: process.env.STORYBOOK_MAPBOX_TOKEN,
+    size: {
+      height: args.height,
+      width: args.width,
+    },
+    startPosition: {
+      long: args.startPositionLong,
+      lat: args.startPositionLat,
+      zoom: args.startPositionZoom,
+    },
+    style: args.style,
+    sourceJson: args.sourceJson,
+    cluster: args.cluster,
+    clusterOptions: {
```

Comment:
maybe args could be grouped into table? https://storybook.js.org/docs/react/writing-docs/doc-block-argstable or somehow format a bit nicer? well it's just asthetics, also is this is how many options we need?